### PR TITLE
Add SPDX-License-Identifier to recipes

### PR DESCRIPTION
**Pull request by GordanM**
Signed-off-by: Gordan Markuš <gordan.markus@pelagicore.com>

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Pelagicore
+Copyright (c) 2017 Pelagicore
```
Comment (rpannek) [inline LICENSE]: I think it should be (c) 2016-207.

File: conf/layer.conf [hunk 2]
```diff
@@ -1,6 +1,8 @@
 #
-#   Copyright (C) 2016 Pelagicore AB
+#   Copyright (C) 2017 Pelagicore AB
```
Comment (rpannek) [inline conf/layer.conf]: I think it should be (c) 2016-207.

Comment (GordanM) [inline conf/layer.conf]: Valid points!

**Comment by GordanM** [comment]
Updated.

File: LICENSE [hunk 3]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Pelagicore
+Copyright (c) 2016-2017 Pelagicore
```
Comment (zeenix) [inline LICENSE]: I'd put this (and the other same change) to a separate patch but it's not a biggie since it's kinda related.

**Comment by GordanM** [comment]
@zeenix could you suggest what to put in the log to make it better?

**Comment by GordanM** [comment]
Updated.

**Comment by zeenix** [comment]
Ah nevermind, it obviously is that. :)

File: LICENSE (hunk 3 above)
Comment (jeremiah) [inline LICENSE]: Copyright is for the life of the author, plus 70 years. So it is not really necessary to have 2016 - 2017. The original year of copyright is sufficient.

Comment (zeenix) [inline LICENSE]: @jeremiah oh, good point. Never thought of it that way. Would have saved me all those updates. :D

**Pull request merged**